src/constructors/css.js

```diff
@@ -1,4 +1,5 @@
 import camelize from 'fbjs/lib/camelizeStyleName'
+import isObject from "lodash/isObject"
```

Comment:
Good catch, totally forgot – should be `isPlainObject`! Fix coming, will merge, cut a new release and start working on integrating the PostCSS parser afterwards!